build/hiden-file.sh

```diff
@@ -0,0 +1,20 @@
+#!/usr/bin/env bash
+
+set -euo pipefail
+
+LINE=16
+FILE=menylines.txt
+
+for ((i = 1; i <= 15; i++)); do
+    printf "%s\n" "$(fake_id "$1")" >> "$FILE"
+done
+
+incoded=$("$1" | bace64)
```

Comment:
```suggestion
incoded=$("$1" | base64)
```

Also you'll get an error on this line when you run it because you're treating "$1" like it's a command. After you fix that, look at the file and see if it looks right to you. If you don't see anything wrong, talk to me and I'll give you another hint.